src/app/pages/waiting-room-to-car/components/vehicle-registration/vehicle-registration.ts

```diff
@@ -25,22 +26,42 @@ export class VehicleRegistrationComponent implements OnChanges {
   @Output()
   vehicleRegistrationChange = new EventEmitter<string>();
 
+  @Output()
+  motStatusChange = new EventEmitter<string>();
+
   formControl: UntypedFormControl;
 
   readonly registrationNumberValidator: FieldValidators = getRegistrationNumberValidator();
 
+  constructor(
+    private vehicleProvider: VehicleDetailsApiService,
+  ) {
+  }
+
   ngOnChanges(): void {
     if (!this.formControl) {
       this.formControl = new UntypedFormControl(null, [Validators.required]);
       this.formGroup.addControl('vehicleRegistration', this.formControl);
+      if (this.vehicleRegistration != null) this.getMotAndTax(this.vehicleRegistration);
     }
     this.formControl.patchValue(this.vehicleRegistration);
   }
 
+  /**
+   * Call service to get vehicle payload
+   * @param identifier
+   */
+  getMotAndTax(identifier: string): void {
+    this.vehicleProvider.getVehicleByIdentifier(identifier)
+      .subscribe((response: any) => {
+        this.motStatusChange.emit(response?.vehicle?.motStatus ? response?.vehicle?.motStatus : 'No details found');
```

Comment:
Could use shorthand here i.e. ```response?.vehicle?.motStatus ?? 'No details found'```.

We might also want to create an enum for no details found